src/flows/components/Sidebar.tsx

```diff
@@ -182,7 +182,10 @@ const Sidebar: FC = () => {
               return true
             }
 
-            if (!/^(inputs|transform)$/.test(PIPE_DEFINITIONS[type]?.family)) {
+            if (
+              !/^(inputs|transform)$/.test(PIPE_DEFINITIONS[type]?.family) &&
+              flow.data.byID[id].type !== 'visualization'
```

Comment:
`type` is already defined above, right? We can change this to `type !== 'visualization'` which improves readability

Reply:
Done